docs/api/algorand/asset-info.mdx

```diff
@@ -0,0 +1,48 @@
+# AssetInfo
+
+Returns informations regarding DeCash assets.
+
+## Request Parameters
+
+|     Name    |                Type               | Description |
+| ----------- | --------------------------------- | ----------- |
+| `assetId`   | OPTIONAL `string`                 | The ID of the DeCash asset of which informations will be returned. If not set, informations about all DeCash tokens will be returned. |
```

Comment:
I would move optional to description; logically it matches there better than in type column